### ISSUE TITLE
Package Size

### ISSUE THREAD

**Issue by strengejacke** [easystats/bayestestR#252]
For the vignettes, we specify "dpi=300", which actually bloats the resulting HTML files (see CRAN). We could reduce the file size of the packages by a substantial amount (and hence, fix the notes in the CRAN check results) when we don't specify the DPI.

What do you think?

**Comment by DominiqueMakowski** [comment]
But then the plots are ugly?

**Comment by strengejacke** [comment]
Hm, isn't screen resolution 72dpi anyway? Or we maybe exclude vignettes from the package and only have them here on the GitHub webpage

**Comment by strengejacke** [comment]
But my feeling is that "lightweight" means little dependencies and small package size...

**Comment by DominiqueMakowski** [comment]
I think it's good that vignettes are also available on CRAN, so I'd prefer to lower the quality rather than remove them altogether. (Personally I am not sure when the memory size of a package can be an issue, so my understanding of lightweight is mostly dependency-wise, but still we must comply with CRAN's limit)

**Comment by strengejacke** [comment]
I just saw that most vignettes already use the default DPI (which is 96, I think), so actually, many vignettes already use the "ugly" plots, e.g. https://cran.r-project.org/web/packages/bayestestR/vignettes/credible_interval.html

We could probably stick to the 96 DPI default, and then lower the image dimensions (instead of 10.08 x 6, as it currently is, maybe to 8x5 or so?)

**Comment by DominiqueMakowski** [comment]
Fine by me, as long as it gets not too pixelated :)

**Comment by strengejacke** [comment]
I changed the DPI in three vignettes from 300 to 150 (https://github.com/easystats/bayestestR/commit/a71a55a173fddfd7a90efa41238ebd91afaed964), and we actually included figures that we don't use no more (https://github.com/easystats/bayestestR/commit/428b797474caf2b882ff09e046a68b76b70dc10d), which I removed.

I will check package size now, and see if we could reduce it by a fair amount. Then I can update the wwbsite, so we can see how plots look like.
